lesson05/assignment/index.html

```diff
@@ -5,11 +5,125 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta http-equiv="X-UA-Compatible" content="ie=edge" />
         <link rel="stylesheet" href="css/style.css" />
-        <title>Assignment 05 - CSS Grid</title>
+        <title>iPhones</title>
     </head>
     <body>
-        <main class="main">
+
+      <!-- grid  -->
+      <div class="grid-container">
+        <header>
+          <h1>iPhones</h1>
+          <p>Keep up with the latest phones!</p>
+        </header>
+
+        <!-- Navigation bar -->
+        <nav class="nav">
+          <ul>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#" class="active">iPhones</a></li>
+            <li><a href="#">Samsungs</a></li>
+            <li><a href="#">Pixels</a></li>
+          </ul>
+        </nav>
+
+        <!-- Sidebar - gives info about brand of phone -->
+        <div class="sidebar">
```

Comment:
I normally just use a div for the sidebar, but `<aside>` might be a better option for accessibility.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside